### PR TITLE
Improve upon close-position PR

### PR DESCRIPTION
**Pull request by luckysori**
Follow-up to https://github.com/get10101/10101/pull/228.

### CODE REVIEW

**Comment by luckysori** [comment]
As discussed, feel free to take over and merge this, @holzeis.

File: crates/trade/src/cfd.rs [hunk 1]
```diff
@@ -3,6 +3,8 @@ use bdk::bitcoin::Denomination;
 use rust_decimal::prelude::ToPrimitive;
 use rust_decimal::Decimal;
 
+pub const BTCUSD_MAX_PRICE: u64 = 1_048_575;
```
Comment (holzeis) [inline crates/trade/src/cfd.rs:6]: nit: I guess we should add a comment here, that this number is not chosen randomly, but is actually a requirement coming from the p2pdoracle we are using at the moment.

🤔 one place were we could put this constant could be on the oracle client, if we choose to wrap it.

File: coordinator/src/node.rs [hunk 2]
```diff
@@ -259,17 +259,16 @@ fn build_contract_descriptor(
 /// function like we used to do in ItchySats.
 fn build_payout_curve(
     total_collateral: u64,
-    execution_price: f64,
+    initial_price: f64,
```
Comment (holzeis) [inline coordinator/src/node.rs:263]: 🤔 I don't think `initial_price` is completely right here, as this function will also be used with the `close_price`. I found `execution_price` to fitting best to reflect both kind of prices.

Note, I agree on the `build_contract_descriptor` function

**Review by holzeis — APPROVED** [review]
Thank you 👍

File: coordinator/src/node.rs (hunk 2 above)
Comment (luckysori) [reply coordinator/src/node.rs:263]: It's true that we use it when closing the contract too, but you better use the `initial_price` to construct the payout function if you want it to be the same one again!

**Pull request merged**